core/assets/bundles/bundle_de.properties

```diff
@@ -1172,8 +1172,8 @@ block.bridge-conveyor.name = Brücken-Förderband
 block.plastanium-compressor.name = Plastanium-Verdichter
 block.pyratite-mixer.name = Pyratit-Mixer
 block.blast-mixer.name = Sprengmixer
-block.solar-panel.name = Solarpanel
-block.solar-panel-large.name = Großes Solarpanel
+block.solar-panel.name = Solarpaneel
```

Comment:
Solarzelle

Reply:
Solarzelle bezieht sich in der Regel auf ein einzelnes Bauelement. Solarpanel oder Solarmodul wäre passender. Die Schreibweise mit doppeltem E ist nicht korrekt.